cg/models/orders/validators/excel_sample_validators.py

```diff
@@ -62,6 +62,10 @@ def convert_to_lower(value: Optional[str]):
     return value.lower()
 
 
+def replace_spaces_with_underscores(value: Optional[str]):
+    return value.replace(" ", "_")
```

Comment:
Don't know if we should keep the inputs to these validators as Optionals given that they most likely will break with no given input. But on the other hand, maybe we want them to break?

Reply:
I think we should remove optional

Reply:
Done 👍